packages/input-datepicker/src/LionInputDatepicker.js

```diff
@@ -306,7 +306,10 @@ export class LionInputDatepicker extends LionInputDate {
 
   async __openCalendarOverlay() {
     this._overlayCtrl.show();
-    await this._calendarElement.updateComplete;
+    await Promise.all([
+      this._calendarOverlayElement.updateComplete,
+      this._calendarElement.updateComplete,
+    ]);
```

Comment:
this is needed to fix `this._calendarElement.focusCentralDate();` in `_onCalendarOverlayOpened`, otherwise it is run before the overlay finished first render and the button within the calendar can not be focused yet since it is not yet in the viewport DOM

not sure how it works in the old karma setup, must be smth with transpilation which could have fixed it somehow, but it is good that we now have it working natively